src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
```

Comment:
Each attribute should start on a new line with 2-space indentation when there are more than two attributes or attribute values are long. This applies to the `<a>` tag on lines 21-24 and the `<img>` tag on lines 25-28.